.gitignore

```diff
@@ -146,10 +147,22 @@ iphone/Maps/app.omaps/
 
 *.autosave
 
+# CMake/ninja
 cmake-build-*
 build/
+.cmake/
+**.cmake
+CMakeFiles/
+CMakeCache.txt
+OMaps*.app/
+build.ninja
+
+# Tests
+Testing/
```

Comment:
All these lines should not be needed if cmake is used properly from a different directory.